app/serializers/activitypub/activity_serializer.rb

```diff
@@ -42,4 +42,8 @@ def atom_uri
   def announce?
     object.reblog?
   end
+
+  def owned_announce?
+    announce? && object.account == object.proper.account
```

Comment:
I am thinking, probably not worth wasting traffic for public items that can be fetched? On the other hand, it does spare the traffic from fetching... But then, status are commonly already known before being boosted, so it's definitely a waste in those cases...

Reply:
I don't know, including the toot is usually pretty cheap compared to performing an HTTPS request. I changed it to only embed private toots for now, but we may benefit from serializing it anyway in general. Indeed, in cases where the toot is not known to remote instances, it would be *much* cheaper to serialize it.